Makefile

```diff
@@ -5,14 +5,24 @@ FULL_GOPATH=${GOPATH}/src/${FULL_NAME}
 
 default: test
 
-get:
-	go get "github.com/omeid/go-resources/cmd/resources"
+dependencies:	
 	go get -t -v ./...
 
-build: get
-	cd example/agent-ping && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
-	cd example/agent-pong && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
+build: dependencies	
 	go build -v ./...
 
+examples: dependencies
```

Comment:
Mmm, I'm pretty sure you have the agent-ping already installed in your go path. Your problem is related with the issue #3 I raised. The agents are not build when simply doing "make".

In my configuration:

$ make
$ ./agent-ping
No such file or directory

$ make examples
$ ./agent-ping
Agent responsible for sending ping commands to the pong agent
...

That's why I added a specific target in the Makefile for building the examples.